src/preamble.js

```diff
@@ -891,6 +891,90 @@ function getBinaryPromise() {
 // Create the wasm instance.
 // Receives the wasm imports, returns the exports.
 function createWasm(env) {
+#if AUTODEBUG
+  env['setTempRet0'] = setTempRet0;
+  env['getTempRet0'] = getTempRet0;
```

Comment:
I thought these two functions lived in wasm and were generated by binaryen?  Maybe fastcomp only?

Reply:
The issue is that the instrumentation passes may add a requirement for those functions, that wasn't there before (since there are the i64 logging calls, which may be the first illegal imports).

Reply:
Hmm.. I was confused I thought binaryen always generated these function on the wasm side.. maybe I was confusing them with stackSave/stackRestore